src/documentation/pages/components/tabs/examples/_vertical-subtabs-lumen-centurylink.vue

```diff
@@ -0,0 +1,117 @@
+<template lang="pug">
+  <ComponentExample title="Vertical Subtabs" :id="exampleId" additionalClasses="-bg--grey-20" padding="-p--0" :tabs="exampleTabs" :headTabs="headTabs" @chiHeadTabsChange="e => changeSelectedTab(e)">
+    .-p--3(slot="example")
```

Comment:
`ComponentExample` makes by default a `div` with a `-p--3` and you are creating another `div` to give it this `-p--3`. Just remove the `padding="-p--0"` from the `ComponentExample` and use the following div as the `slot="example`

```suggestion
  <ComponentExample title="Vertical Subtabs" :id="exampleId" additionalClasses="-bg--grey-20" :tabs="exampleTabs" :headTabs="headTabs" @chiHeadTabsChange="e => changeSelectedTab(e)">
```

Check this in every example.